source/server/hot_restarting_parent.cc

```diff
@@ -98,7 +98,10 @@ void HotRestartingParent::exportStatsToChild(HotRestartMessage::Reply::Stats* st
   for (const auto& counter : server_->stats().counters()) {
     // The hot restart parent is expected to have stopped its normal stat exporting (and so
     // latching) by the time it begins exporting to the hot restart child.
-    (*stats->mutable_counter_deltas())[counter->name()] = counter->latch();
+    int latched_value = counter->latch();
```

Comment:
nit: uint64_t. Also, did you add a test case for this?

Reply:
Done. Added a test for that, as well as some of the pre-existing hot restart parent logic.